.github/workflows/release-cli.yml

```diff
@@ -132,13 +132,14 @@ jobs:
         uses: matthiaszepper/sphinx-action-v4@latest
         with:
           container: latest
-          pre-build-command: "pip install ."
+          pre-build-command: "apt-get update && apt-get install -y gcc && pip install ."
           build-command: "make html"
           docs-folder: "docs/"
       - name: Build the pdf with Sphinx
         uses: matthiaszepper/sphinx-action-v4@pdflatex
         with:
           container: pdflatex
+          pre-build-command: "apt-get update && apt-get install -y librsvg2-bin"
```

Comment:
The building was failling because the machine needed those dependencies. gcc in the case of html and the modules to convert the svg in the case of the pdf. 

Maybe there was some update of the actions that is importing and that's why those libraries were no longer included? For the pdf, I suspect some update is what break it, this post is what helped me debug it
https://stackoverflow.com/questions/49898798/using-sphinx-extension-to-convert-svg-to-pdf